[Add-ins] Implement feedback on overview and quick start (#625)
OfficeDev/outlook-dev-docs: docs/add-ins/quick-start.md

title: Build an Outlook task pane add-in
description: Learn how to build a simple Outlook add-in by using jQuery and the Office JS API and test it locally.
ms.topic: quickstart
ms.date: 06/13/2019
localization_priority: Priority
---

# Build your first Outlook add-in

In this article, you'll walk through the process of building an Outlook task pane add-in that displays at least one property of a selected message.

## Create the add-in

> You can create an Office Add-in by using Visual Studio or any other editor. Tell us what editor you'd like to use by choosing one of the following tabs.

Reviewer: ```suggestion
You can create an Office Add-in by using Visual Studio or any other editor. Tell us which editor you prefer by choosing one of the following tabs.
```

1. In the **Connect to Exchange email account** dialog box, enter the email address and password for your [Microsoft account](https://account.microsoft.com/account) and then choose **Connect**. When the Outlook.com login page opens in a browser, login to your email account with the same credentials as you entered previously.

    > [!NOTE]

>     > If the **Connect to Exchange email account** dialog box repeatedly prompts you to log in, Basic Auth may be disabled for accounts on your Office 365 tenant. To test this add-in, login using a [Microsoft account](https://account.microsoft.com/account) instead.

Reviewer: ```suggestion
    > If the **Connect to Exchange email account** dialog box repeatedly prompts you to sign in, Basic Auth may be disabled for accounts on your Office 365 tenant. To test this add-in, login using a [Microsoft account](https://account.microsoft.com/account) instead.
```
We always use "sign in" and "sign out" unless the dialog really does use the text "log in".
